docs/source/examples/replicate_vicuna/chatbot.py

```diff
@@ -1,32 +1,43 @@
+import argparse
 import gradio as gr
 import numpy as np
-import os
-import sentencepiece as spm
 import time
 
-import ctranslate2
-
-from onmt.utils.logging import init_logger
-from onmt.translate.translator import build_translator
-from onmt.inputters.text_utils import textbatch_to_tensor
-from onmt.inputters.inputter import IterOnDevice
-from onmt.transforms import get_transforms_cls, TransformPipe
-from onmt.transforms import make_transforms
 import onmt.opts as opts
+from onmt.transforms.tokenize import SentencePieceTransform
 from onmt.utils.parse import ArgumentParser
 from onmt.utils.misc import use_gpu, set_random_seed
 
-inf_type = "-py"
-# inf_type = "ct2"
+parser = argparse.ArgumentParser()
+parser.add_argument(
+    "--inference_config_file",
+    "-inference_config_file",
```

Comment:
Either do this "--"/"-" duality for all flags, or none, but doing it only for one is misleading.